### PR TITLE
Removed Typings dependenciy in favor of @types

### PR DESCRIPTION
**Pull request by con-ssc**

### CODE REVIEW

File: lib/dependencies.js [hunk 1]
```diff
@@ -46,7 +46,10 @@ let versionMap = {
   "minimatch": "^3.0.2",
   "through2": "^2.0.1",
   "tslint": "^3.11.0",
-  "typings": "^1.3.0",
+  "@types/protractor": "^1.5.20",
+  "@types/jasmine": "^2.5.35",
+  "@types/selenium-webdriver": "^2.53.33",
```
Comment (Ullfis) [inline lib/dependencies.js]: Could you test this?

In my projects I have to lock this to version `"@types/selenium-webdriver": "2.44.26",`

**Comment by EisenbergEffect** [comment]
@niieani Can you review this?

File: lib/dependencies.js [hunk 2]
```diff
@@ -46,7 +46,10 @@ let versionMap = {
   "minimatch": "^3.0.2",
   "through2": "^2.0.1",
   "tslint": "^3.11.0",
-  "typings": "^1.3.0",
+  "@types/protractor": "^1.5.0",
+  "@types/jasmine": "^2.2.0",
+  "@types/selenium-webdriver": "^2.44.0",
```
Comment (niieani) [inline lib/dependencies.js]: Protractor 4 comes with its own typings and they contain jasmine and selenium. No need to even add these three nowadays. The rest is okay.

Comment (con-ssc) [inline lib/dependencies.js]: I can't see that protractor is installed with angular-cli yet. If you could point me at the file where the dependency to protractor is stated.

**Comment by EisenbergEffect** [comment]
@niieani Do we need to do modifications here or is this good to go?

**Comment by niieani** [comment]
@EisenbergEffect the question is whether we want to install typings for Protractor / Selenium / Jasmine by default or not. This is what CLI was doing with the old-style Typings. If the answer is no, then this should be fine to merge.

**Comment by EisenbergEffect** [comment]
For Jasmine we want to install them if they selected the testing option in a TS setup. Otherwise, we don't.

**Comment by niieani** [comment]
In that case OP will want to modify the PR so that adding a dependency on `@types/jasmine` is conditional. Otherwise remove `@types/protractor` and `@types/selenium-webdriver` dependencies and this is good.

**Comment by milkshakeuk** [comment]
@con-ssc is this something you are still working on?

**Comment by con-ssc** [comment]
@milkshakeuk Not really.

File: lib/commands/new/transpilers/typescript.js [hunk 3]
```diff
@@ -24,11 +20,6 @@ module.exports = function(project) {
     'gulp-typescript',
     'gulp-tslint',
     'tslint',
-    'typescript',
```
Comment (milkshakeuk) [inline lib/commands/new/transpilers/typescript.js]: shouldn't `@types/jasmine` and `@types/node` be added to the list of dev dependencies here? for them to be included in the `project.json` ready for when `installDependencies` is called in `project-template.js`?

Comment (niieani) [inline lib/commands/new/transpilers/typescript.js]: devDependencies, yes

**Comment by EisenbergEffect** [comment]
Merged #457

**Pull request closed without merging**